slemicro/create_vm_with_eib.sh

```diff
@@ -100,15 +129,15 @@ if [ $(uname -o) == "Darwin" ]; then
 	# If there are not extra disks, this works as well
 	UTMDISKMAPPING="{removable:false, source:rootfs}${UTMEXTRADISKMAPPING}}"
 
-	# Create and launch the VM using UTM
+	# Create the VM
 	OUTPUT=$(osascript <<-END
 	tell application "UTM"
 		-- specify the RAW file
 		set rootfs to POSIX file "${VMFOLDER}/${VMNAME}.qcow2"
 		-- specify extra disks
 		${UTMEXTRADISKS}
 		--- create a new QEMU VM
-		set vm to make new virtual machine with properties {backend:qemu, configuration:{cpu cores:${CPUS}, memory: ${MEMORY}, name:"${VMNAME}", architecture:"aarch64", drives:${UTMDISKMAPPING}}
+		set vm to make new virtual machine with properties {backend:qemu, configuration:{cpu cores:${CPUS}, memory: ${MEMORY}, name:"${VMNAME}", network interfaces:{{hardware:"virtio-net-pci", mode:shared, index:0, address:"${MACADDRESS}", port forwards:{}, host interface:""}}, architecture:"aarch64", drives:${UTMDISKMAPPING}}
```

Comment:
if mac value is null by default this part is going to fail ???

Reply:
The https://github.com/suse-edge/misc/pull/109/files#diff-1fe767c7e29ef34f346230be6794a5513ee46bdcc30a65713ccf78b29fdbddfdR72 check will fail before all that